demos/layout-grid.html

```diff
@@ -0,0 +1,178 @@
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="assets/material-components-web.css.js" charset="utf-8"></script>
+
+    <style>
+    .mdc-layout-grid {
+      background-color: #DDDDDD;
+      margin-bottom: 32px;
+    }
+
+    .mdc-layout-grid__cell {
+      box-sizing: border-box;
+      background-color: #666666;
+      height: 200px;
+      padding: 8px;
+      color: white;
+
+      font-size: 1.5em;
+    }
+    .mdc-layout-grid:first-of-type .mdc-layout-grid__cell {
+      height: 50px;
+    }
+
+    .mdc-layout-grid.max-width {
+      max-width: 1600px;
+    }
+
+    .mdc-layout-grid.left-align {
+      margin-left: 0;
+      margin-right: auto;
+    }
+
+    .grid-legend {
+      margin: 16px 0 8px 0;
+    }
+
+    .ruler {
+      position: fixed;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      bottom: 0;
+      left: 0;
+      height: 20px;
+      width: 100%;
+      margin: 0;
+      background: black;
+      color: white;
+    }
+    </style>
+  </head>
+  <body>
+    <main class="mdc-typography">
+      <h1>MDC Layout Grid</h1>
+
+      Margin:
+      <select id="margin">
+        <option value="8px">8px</option>
+        <option value="16px" selected>16px</option>
+        <option value="24px">24px</option>
+        <option value="40px">40px</option>
+      </select>
+
+      <br>
+
+      Gutter:
+      <select id="gutter">
+        <option value="8px">8px</option>
+        <option value="16px" selected>16px</option>
+        <option value="24px">24px</option>
+        <option value="40px">40px</option>
+      </select>
+
+
+      <div class="grid-legend">Grid of 1 column wide items</div>
+      <div class="mdc-layout-grid">
+        <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1">1</div>
+        <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1">1</div>
+        <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1">1</div>
+        <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1">1</div>
+        <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1">1</div>
+        <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1">1</div>
+        <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1">1</div>
+        <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1">1</div>
+        <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1">1</div>
+        <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1">1</div>
+        <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1">1</div>
+        <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1">1</div>
+      </div>
+
+      <div class="grid-legend">Grid of 4 column wide items</div>
+      <div class="mdc-layout-grid">
+        <div class="mdc-layout-grid__cell mdc-layout-grid__cell--span-4 mdc-layout-grid__cell--order-1">5</div>
```

Comment:
Oops, that was some random testing that got committed. Not supposed to be there. I'll leave the ordering stuff for the readme.